### PR TITLE
port dsd image integration tests to python to remove docker binary dep

### PR DESCRIPTION
**Pull request by xvello**
### What does this PR do?

Port the dsd image integration test from bash + docker cli to python + docker-py

### Motivation

Run it on CCI, slimmer runner & better error handling

### CODE REVIEW

File: test/integration/docker/dsd_listening.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    global containers
+    global client
+
+    for _, container in containers.iteritems():
+        container.stop()
+
+
+def waitUntilListening(container, retries=20):
+    for x in range(0, retries):
+        out = container.exec_run(cmd="netstat -a")
```
Comment (CharlyF) [inline test/integration/docker/dsd_listening.py]: Why using netstat instead of lsof ?

**Review by CharlyF — COMMENTED** [review]
Just one question, looks good otherwise

File: test/integration/docker/dsd_listening.py (hunk 1 above)
Comment (xvello) [reply test/integration/docker/dsd_listening.py]: netstat is already shipped in alpine, while lsof had to be installed. this shaves off the install time from the test time

Comment (CharlyF) [reply test/integration/docker/dsd_listening.py]: Smart.

**Review by CharlyF — APPROVED** [review]
🐑 🇮🇹

**Pull request merged**